zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/authorization/DbRoleState.java

```diff
@@ -54,5 +58,10 @@ public DbRoleState(
     roleByNameColumnFamily =
         zeebeDb.createColumnFamily(
             ZbColumnFamilies.ROLE_BY_NAME, transactionContext, roleName, fkRoleKey);
+
+    fkTenantKey = new DbForeignKey<>(roleKey, ZbColumnFamilies.ROLES);
+    tenantByIdColumnFamily =
+        zeebeDb.createColumnFamily(
+            ZbColumnFamilies.TENANT_BY_ID, transactionContext, tenantId, fkTenantKey);
```

Comment:
❌ This column family should be moved to the `DbTenantState`. The foreign key shouldn't refer to the `ROLES` column family, but to the `TENANTS` column family.

Reply:
I see I missed to change this when copy-pasting to this issue 😄 I've changed it in the issue as well.